### PR TITLE
1. User Authentication

### PR DESCRIPTION
**Pull request by MaxRazen**
## Description
https://docs.google.com/document/d/1YxhvvTMYvrLUe_Mtta67i9VzawW1EeqtlGv3TH1bF3I/edit

### CODE REVIEW

File: app/app/Http/Requests/Auth/LoginRequest.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+
+    /**
+     * Attempt to authenticate the request's credentials.
+     *
+     * @return void
+     *
+     * @throws \Illuminate\Validation\ValidationException
+     */
+    public function authenticate()
```
Comment (vsobchuk) [inline app/app/Http/Requests/Auth/LoginRequest.php]: @MaxRazen please take into account that placing logic in Request is a violation of SOLID principles. Please use Service class to store application logic. It is true for all the methods within all the classes that are not related to what that class aimed for.

Comment (MaxRazen) [reply app/app/Http/Requests/Auth/LoginRequest.php]: @vsobchuk good catch, agree
Do I also need to re-struct the app to DDD?

Comment (MaxRazen) [reply app/app/Http/Requests/Auth/LoginRequest.php]: @vsobchuk https://github.com/MaxRazen/cookbook-demo/pull/2/commits/02b98e5fc90d43067d4060a5ba19fc860efcd76a
